src/components/ec-currency-input/ec-currency-input.spec.js

```diff
@@ -106,6 +112,40 @@ describe('EcCurrencyInput', () => {
     });
   });
 
+  describe('when the placeholder message for the amount input is defined', () => {
+    it('should render properly', () => {
+      const wrapper = mountCurrencyInput({ amountPlaceholder: 'Amount Placeholder' });
+      expect(wrapper.findByDataTest('ec-currency-input__amount').element).toMatchSnapshot();
+    });
+  });
+
+  describe('when the disabled currency selector has a tooltip associated', () => {
+    it('should render properly', () => {
+      const tooltipMessage = 'Tooltip message';
+      const wrapper = mountCurrencyInput({
+        currenciesTooltipInfo: { content: tooltipMessage },
+        isCurrenciesDisabled: true,
+      });
+      expect(wrapper.findByDataTest('ec-currency-input__currencies').element).toMatchSnapshot();
+      expect(wrapper.findByDataTest('ec-currency-input__currencies').attributes('mocked-tooltip-content')).toBe(tooltipMessage);
+      expect(wrapper.findByDataTest('ec-currency-input__currencies').attributes('mocked-tooltip-placement')).toBe('top');
```

Comment:
You have tested both of those in `.toMatchSnapshot();`;

Reply:
Duplicated tests removed.